src/content/docs/accounts/accounts-billing/account-setup/choose-your-data-center.mdx

```diff
@@ -41,10 +41,10 @@ New Relic offers almost all the same active products, features, support offering
 
 **Exceptions:** The following are **not supported** with an EU region account:
 
-* [Infinite Tracing](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing) is not available.
-* APM's [weekly performance reports](/docs/apm/reports/other-performance-analysis/weekly-performance-report) are not available.
+* [Infinite Tracing](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing)
+* APM's [weekly performance reports](/docs/apm/reports/other-performance-analysis/weekly-performance-report)
 * [Log patterns](/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns/#availability)
-* Deprecated products and features are not available.
+* Deprecated products and features
 
 [New Relic's Incident Intelligence](/docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-incident-intelligence) service operates solely in the US whether you store your data in New Relic’s US region data center or New Relic’s EU region data center, by using New Relic Incident Intelligence, you consent that New Relic may move and store your data in the US region.
```

Comment:
line 49 needs to be broken into 2 sentences. your call whether to break at "...operates solely in the US. Whether you store..." or at "...EU region data center. By using New Relic Incident Intelligence..."